test/project.spec.ts

```diff
@@ -6,17 +6,17 @@ import { tsquery } from '../src/index';
 
 describe('tsquery:', () => {
     describe('tsquery.project:', () => {
```

Comment:
I had a problem with the tests below. I don't know why but some times when I was launching the test the `file.length` was incrementing even if I didn't touch the `./tsconfig.json`. Do I need to open an issue ?